lib/quadquizaminos_web/live/contests_live.ex

```diff
@@ -159,10 +157,20 @@ defmodule QuadquizaminosWeb.ContestsLive do
     {:noreply, socket |> assign(current_user: current_user, current_uri: uri)}
   end
 
-  defp _create_contest(socket, contest_name) do
+  defp _create_contest(socket, contest_name, "") do
```

Comment:
You can pass the params received as the second argument , then pattern match the date
```suggestion
  defp _create_contest(socket, %{"contest_date" => ""}= params) do
  params = %{params | "contest_date" => nil}
  case Contests.create_contest(params) do
      {:ok, contest} -> assign(socket, contests: contests ++ [contest])
      _ -> socket
    end
```